pkg/plugins/platform/local.go

```diff
@@ -974,6 +986,57 @@ func GetActiveApp(siteName string) (App, error) {
 		return app, err
 	}
 
-	err = app.Init(activeAppRoot)
-	return app, err
+	_ = app.Init(activeAppRoot)
```

Comment:
no err check?

Reply:
Thanks, added a comment on why it's ignored (app.Init() fails if directory is gone, and we already know that we have a real app)